springfield/settings/__init__.py

```diff
@@ -22,17 +22,64 @@
 # CSP settings, expanded upon later:
 # NOTE: We are providing all settings to django-csp as sets, not lists.
 # - This is for de-duping, and because django-csp will convert them to `sorted` lists for us.
+
+# NOTE: Any URLs that contain a path, not just the origin, trailing slashes are important.
+# - if no path is provided, all resources are allowed from the origin.
+# - if path is provided with no trailing slash: an exact-match is required.
+#   - e.g. `https://example.com/api` will only match `https://example.com/api`
+# - if path is provided with trailing slash: the path is a prefix-match.
+#   - e.g. `https://example.com/api/` will match anything that starts with `https://example.com/api/`
+
 _csp_default_src = {
+    # Keep `default-src` minimal. Best to set resources in the specific directives.
     csp.constants.SELF,
-    "*.firefox.com",
-    "assets.mozilla.net",
+}
+_csp_connect_src = {
+    # NOTE: Check if these need to be in the `_csp_form_action` list as well since we often
+    # progressively enhance forms by using Javascript.
+    csp.constants.SELF,
+    BASKET_URL,
+    "www.googletagmanager.com",
+    "www.google-analytics.com",
+    "region1.google-analytics.com",
+    "o1069899.sentry.io",
+    "o1069899.ingest.sentry.io",
+    FXA_ENDPOINT,  # noqa: F405
+}
+_csp_font_src = {
+    csp.constants.SELF,
+}
+_csp_form_action = {
+    csp.constants.SELF,
+    # NOTE: Check if these need to be in the `_csp_connect_src` list as well since we often
+    # progressively enhance forms by using Javascript.
```

Comment:
Can we check this before we merge?

Reply:
That comment is more for future devs. I've checked it for what's there now.